mifosng-android/build.gradle

```diff
@@ -142,8 +143,8 @@ dependencies {
     implementation fileTree(dir: 'src/main/libs', include: ['*.jar'])
 
     //DBFlow dependencies
+    annotationProcessor "com.github.Raizlabs.DBFlow.dbflow:dbflow-processor:3.1.1"
```

Comment:
@droidchef  Yes, because initially while building the project , we all were getting the annotation processor error , though the annotation settings were set to be true.

Reply:
```
       annotationProcessor "com.github.Raizlabs.DBFlow:dbflow-processor:$rootProject.raizLabsDBFlow"
       annotationProcessor "com.github.Raizlabs.DBFlow.dbflow:dbflow-processor:3.1.1"
   ```
You see, we have two `annotationProcessor` lines trying to fetch the same thing. I can imagine one of them is not working and that's why, I am asking do we need both, if not, can you remove the wrong/redundant one, please :)

Reply:
@droidchef I have removed the redundant one , kindly review . : )

Reply:
@droidchef Please have a look.